Makefile

```diff
@@ -16,18 +16,10 @@ image-build:
 	docker build \
 		--build-arg GOLANG_VERSION=$(GOLANG_VERSION) \
 		--build-arg GOBORING_BUILD=$(GOBORING_BUILD) \
-		--tag $(ORG)/hardened-build-base:$(TAG) \
 		--tag $(ORG)/hardened-build-base:$(TAG)-$(ARCH) \
-		.
+		. \
+		-f Dockerfile.$(ARCH)
 
 .PHONY: image-push
 image-push:
 	docker push $(ORG)/hardened-build-base:$(TAG)-$(ARCH)
-
-.PHONY: image-manifest
-image-manifest:
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create --amend \
-		$(ORG)/hardened-build-base:$(TAG) \
-		$(ORG)/hardened-build-base:$(TAG)-$(ARCH)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push \
-		$(ORG)/hardened-build-base:$(TAG)
```

Comment:
This bit combined with the `ORG` envvar in the `Makefile` makes it handy and easy to test building downstream dependents pointing to your personal/alternate DockerHub repo. Please do not remove it.

Reply:
done in commit https://github.com/rancher/image-build-base/pull/12/commits/62642601dea30e63b8bedba7bb1e8de87b051c4d